packages/main/src/ColorPalette.ts

```diff
@@ -357,6 +370,11 @@ class ColorPalette extends UI5Element {
 	_onColorContainerKeyDown(e: KeyboardEvent) {
 		const target = e.target as ColorPaletteItem;
 		const lastElementInNavigation = this.colorPaletteNavigationElements[this.colorPaletteNavigationElements.length - 1];
+
+		if ((isUp(e) || isDown(e)) && this._currentlySelected !== undefined && this.colorPaletteNavigationElements.includes(this._currentlySelected)) {
```

Comment:
this._currentlySelected !== undefined isn't that the same as !this._currentlySelected? (It's shorter and easier to read.